README.md

```diff
@@ -31,3 +31,84 @@ Find our blog post series [here](https://security.googleblog.com/2022/04/improvi
 ### Specifications
 
 For a more in-depth technical dive, read the [SPECIFICATIONS.md](./SPECIFICATIONS.md).
+
+### Format
+
+Each of the projects utilize the same base GitHub workflow SLSA provenance. The common fields of the SLSA provenance predicate attested to are below.
+
+
+_Added v1.0.0_
+
+`BuildType`: This is the URI for the particular provenance builder, for example, the go or generic builder. 
+```json
+  "buildType": "https://github.com/slsa-framework/slsa-github-generator-go@v1"
+```
+`Builder.Id`: The builder ID refers to the fully qualified URI of the trusted builder's workflow and its reference. 
+```json
+ "builder": {
+      "id": "https://github.com/slsa-framework/slsa-github-generator/.github/workflows/builder_go_slsa3.yml@refs/tags/v0.0.1"
+  }
+```
+`Invocation`: Identifies the event that kicked off the build. This describes the workflow run and includes GitHub workflow event information, entrypoint, and parameters from trigger events. 
+`Invocation.configSource`: This describes the calling workflow's source and the entrypoint of the build.
+```json
+"configSource": {
+    "uri": "git+https://github.com/laurentsimon/slsa-verifier-test-gen@refs/heads/main",
+    "digest": {
+      "sha1": "15bf79ea9c89fffbf5dd02c6b5b686b291bfcbd2"
+    },
+    "entryPoint": "Go SLSA Release"
+},
+```
+`Invocation.parameters`: This describes any parameters from trigger events.
+
+`Invocation.environment`: This describes the GitHub workflow environment, including the event information.
+```json
+"environment": {
+    "arch": "X64",
+    "github_actor": "laurentsimon",
```

Comment:
shall we add some comments about each field here?
I also added support repository_id, repository_owner_id and actor_id.

Reply:
done